virttest/libvirt_vm.py

```diff
@@ -2783,23 +2783,28 @@ def install_package(self, name, ignore_status=False):
                 query_cmd = "dpkg -l | grep %s" % name
                 cmd = "apt install -y %s" % name
                 update_cmd = "apt upgrade -y %s" % name
+                clean_cmd = "apt-get clean"
             else:
                 query_cmd = "rpm -q %s" % name
                 cmd = "yum install -y %s" % name
                 update_cmd = "yum update -y %s" % name
+                clean_cmd = "yum clean all"
 
             if session.cmd_status(query_cmd):
                 # Install the package if it does not exists
                 status, output = session.cmd_status_output(cmd, timeout=300)
                 # Just check status is not enough
                 # It's necessary to check if install successfully
                 if status != 0 or session.cmd_status(query_cmd) != 0:
-                    if not ignore_status:
-                        raise virt_vm.VMError("Installation of package %s "
-                                              "failed:\n%s" % (name, output))
-                    else:
-                        logging.error("Installation of package %s failed:"
-                                      "\n%s", name, output)
+                    session.cmd_status_output(clean_cmd, timeout=300)
```

Comment:
please add one comment here why we need do yum clean .

Reply:
Done